src/AutoAllocation/containers/Unallocated.js

```diff
@@ -20,6 +20,10 @@ class UnallocatedContainer extends Component {
 
   async componentWillMount () {
     try {
```

Comment:
read-only user should be abl to see unallocated list shouldnt they?

Reply:
The reasoning when me and Mike wrote the ticket is that you could get the same information on the manual allocation screen, making the  auto allocation pointless in the context of read only.